src/back-end/lib/db/proposal/sprint-with-us.ts

```diff
@@ -158,6 +159,28 @@ async function rawSWUProposalToSWUProposal(connection: Connection, session: Sess
     return result;
   }));
 
+  // Populate team member status for each phase
+  for (const member of inceptionPhase?.members || []) {
+    member.pending = (await connection('affiliations')
+      .where({ user: member.member.id, organization: organizationId })
+      .select('membershipStatus')
+      .first())?.membershipStatus === MembershipStatus.Pending;
+  }
+
+  for (const member of prototypePhase?.members || []) {
+    member.pending = (await connection('affiliations')
+      .where({ user: member.member.id, organization: organizationId })
+      .select('membershipStatus')
+      .first())?.membershipStatus === MembershipStatus.Pending;
+  }
+
+  for (const member of implementationPhase.members) {
+    member.pending = (await connection('affiliations')
+      .where({ user: member.member.id, organization: organizationId })
+      .select('membershipStatus')
+      .first())?.membershipStatus === MembershipStatus.Pending;
+  }
```

Comment:
Will the use of `first` here incorrectly return `INACTIVE` affiliations if there are any?

Reply:
Yes, nice catch.  Updated with a more restrictive where clause.